lock-api/src/main/java/com/palantir/lock/client/UnreliableTimeLockService.java

```diff
@@ -144,4 +170,45 @@ public void tryUnlock(Set<LockToken> tokens) {
     public long currentTimeMillis() {
         return delegate.currentTimeMillis();
     }
+
+    private void maybeRandomlyIncreaseTimestamp() {
+        buggify.maybe(0.1).run(this::randomlyIncreaseTimestamp);
```

Comment:
we have two hardcoded probabilities in this class - should these be in a param/factory/immutable for easier configuration (or connect to however we configure antithesis runs - this number looks reasonable as a start but we are very probably going to need customization)